src/main/proto/event.proto

```diff
@@ -35,118 +36,229 @@ service EventStore {
     }
 
     // Retrieves the first token available in event store (typically 0). Returns 0 when no events in store.
-    rpc GetFirstToken (GetFirstTokenRequest) returns (TrackingToken) {}
+    rpc GetFirstToken (GetFirstTokenRequest) returns (TrackingToken) {
+    }
 
     // Retrieves the last committed token in event store. Returns -1 when no events in store.
-    rpc GetLastToken (GetLastTokenRequest) returns (TrackingToken) {}
+    rpc GetLastToken (GetLastTokenRequest) returns (TrackingToken) {
+    }
 
     // Retrieves the token of the first token of an event from specified time in event store. Returns -1 when no events in store.
-    rpc GetTokenAt (GetTokenAtRequest) returns (TrackingToken) {}
+    rpc GetTokenAt (GetTokenAtRequest) returns (TrackingToken) {
+    }
 }
 
+/* Request message to receive the first Token (Tail Token) of the Event Stream */
 message GetFirstTokenRequest {
 }
 
+/* Request message to receive the last Token (Head Token) of the Event Stream */
 message GetLastTokenRequest {
 }
 
+/* Request message to receive the Token that starts streaming events from the given timestamp */
 message GetTokenAtRequest {
+
+    /* Timestamp expressed as milliseconds since epoch */
     int64 instant = 1;
 }
 
+/* Message containing the information necessary to track the position of events in the Event Stream */
 message TrackingToken {
+
+    /* The value of the Token */
     int64 token = 1;
 }
 
+/* Message wrapping an Event and a Tracking Token */
 message EventWithToken {
+
+    /* The Token representing the position of this Event in the Stream */
     int64 token = 1;
+
+    /* The actual Event Message */
     Event event = 2;
 }
 
+/* Message providing the parameters for executing a Query against AxonServer. */
 message QueryEventsRequest {
+
+    /* The query to execute against the Event Stream */
     string query = 1;
+
+    /* The number of results AxonServer may send before new permits need to be provided */
     int64 number_of_permits = 2;
+
+    /* Whether to keep the query running against incoming events once the Head of the Stream is reached */
     bool live_events = 3;
 }
 
+/* A message describing the a response to a Query request */
```

Comment:
Typo : `A message describing a response to a Query request` i.o. `A message describing the a response to a Query request`